ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Initiator.java

```diff
@@ -123,11 +124,13 @@ public void run() {
 
           ShowCompactResponse currentCompactions = txnHandler.showCompact(new ShowCompactRequest());
 
+          Set<String> skipDBs = new HashSet<>();
+          Set<String> skipTables = new HashSet<>();
```

Comment:
IIUC compaction should resume on dbs and tables after first successful incremental load. How will these sets be updated to reflect that?

Reply:
Set contents are valid only for a iteration of "txnHandler.findPotentialCompactions". From the second iteration, sets are auto reset and starts computing based on successful incremental loads.

Reply:
Gotcha, thanks!